dev-lang/perl/patches/perl-5.34.0.patchset

```diff
@@ -0,0 +1,406 @@
+From 3410e7e2e37488e45d2914ca6cf02b6d1dbe4584 Mon Sep 17 00:00:00 2001
```

Comment:
I removed all sitecustomize patchs, the removal of hints/ (I use it), the configure patchs (they are merged upstream)